chasm/lib/activity/handler.go

```diff
@@ -16,9 +17,36 @@ func newHandler() *handler {
 }
 
 func (h *handler) StartActivityExecution(ctx context.Context, req *activitypb.StartActivityExecutionRequest) (*activitypb.StartActivityExecutionResponse, error) {
+	request := req.GetFrontendRequest()
+	_, key, _, err := chasm.NewEntity(
+		ctx,
+		chasm.EntityKey{
+			NamespaceID: req.GetNamespaceId(),
+			BusinessID:  request.GetActivityId(),
+		},
+		func(mutableContext chasm.MutableContext, _ any) (*Activity, any, error) {
+			newActivity, err := NewStandaloneActivity(mutableContext, request)
+			if err != nil {
+				return nil, nil, err
+			}
+
+			err = TransitionScheduled.Apply(newActivity, mutableContext, nil)
+			if err != nil {
+				return nil, nil, err
+			}
+
+			return newActivity, nil, nil
+		}, nil)
+
+	if err != nil {
+		return nil, err
+	}
+
 	return &activitypb.StartActivityExecutionResponse{
 		FrontendResponse: &workflowservice.StartActivityExecutionResponse{
-			// TODO
+			Started: true,
+			RunId:   key.EntityID,
+			// EagerTask: TODO need to figure out how to populate this if tasks are async
```

Comment:
Eager task may not make it into the MLP, it's low priority. But really easy to implement since you essentially just need to call the same code that would handle the RecordActivityTaskStarted API.